internal/model/status.go

```diff
@@ -0,0 +1,45 @@
+package model
+
+// 0-NEW, 1-PROCESSING, 2-INVALID, 3-PROCESSED, 4-REGISTERED.
+type Status int
```

Comment:
Если сделать статус стрингой (по сути получим енам) то это снимет довольно много сложностей ниже - все свитч кейсы станут не нужны
```suggestion
type Status string
```

Reply:
Да, что-то я тут слишком сложно сделал, так намного проще все стало.